test/sanitize-html.test.js

```diff
@@ -145,7 +146,12 @@ describe('Sanitize HTML', () => {
   });
 
   test('twitch.tv embedded content should not be removed', () => {
-    const data = sanitizeHTML('<iframe src="https://www.twitch.tv/0pensrc"></iframe>');
-    expect(data).toBe('<iframe src="https://www.twitch.tv/0pensrc"></iframe>');
+    // The parent domain also embeds the stream, so include the web url used here as well
+    const data = sanitizeHTML(
+      `<iframe src="https://player.twitch.tv/?video=1303803789&parent=${WEB_URL}"></iframe>`
```

Comment:
Is this ready to be reviewed? You will need to [transfrom](https://github.com/apostrophecms/sanitize-html#transformations) an iframe we get from dev.to or other websites, remove the parent attribute on it, and add our own following Dave's suggestion.